CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.8.0...master
 
+### Fixed
+- Disable backup service during the test. Resolve [JPERF-344].
```

Comment:
What's "the test"?

The diff shows that we're *adding* `WebJira.configureBackupPolicy` and `BackupConfiguration.disable`.

Reply:
Due to not enabled this feature by default I will change the message to 
`### Added
- Add ability to delete service backup. Resolve [JPERF-334].`

Reply:
Sounds good!